app/models/group.rb

```diff
@@ -22,6 +22,15 @@ def self.all(username = nil)
     groups
   end
 
+  def with_approval_role?
```

Comment:
@hsong-rh Could this be redone based on this issue https://github.com/RedHatInsights/insights-rbac/issues/115
To use the role names directly

Reply:
I don't think so, RedHatInsights/insights-rbac#115 will return roles based on users. Here I need check specific approval roles for groups.